conntrollers/controller.js

```diff
@@ -0,0 +1,11 @@
+class Controller {
```

Comment:
Could you correct `coNNtrollers` folder name?

Reply:
ok